src/Adapter/Currency/CommandHandler/AbstractCurrencyHandler.php

```diff
@@ -122,6 +137,87 @@ protected function applyPatternTransformations(Currency $entity, array $localize
         $entity->setLocalizedPatterns($localizedPatterns);
     }
 
+    /**
+     * @param Currency $entity
+     * @param AddCurrencyCommand $command
+     *
+     * @throws CannotCreateCurrencyException
+     * @throws LanguageNotFoundException
+     * @throws LocalizationException
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
+    protected function createEntity(Currency $entity, AddCurrencyCommand $command)
```

Comment:
As this function is taking the entity as parameter and is not "creating" it itself, maybe the name could be `addEntity` or `saveEntity`? Wdyt?

Reply:
`saveEntity` :+1:

Reply:
Yeah you're right I prefer `addEntity` to emphasize the difference with `updateEntity` Because in object model `save` is a mix of `add/update`

Reply:
Done